src/bootstrap/build/sanity.rs

```diff
@@ -79,7 +79,7 @@ pub fn check(build: &mut Build) {
         }
 
         // Make sure musl-root is valid if specified
-        if target.contains("musl") {
+        if target.contains("musl") && target.contains("x86_64") {
```

Comment:
Hmm, we don't provide binaries releases of std for that target (yet). But I guess we probably want to use static linking for that target as well because is not memory constrained like the mips are. @alexcrichton ?